fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/expressions/Comparisons.java

```diff
@@ -1339,8 +1339,7 @@ public static class ListComparison implements Comparison {
         @Nullable
         private final Descriptors.FieldDescriptor.JavaType javaType;
 
-        @SuppressWarnings({"rawtypes", "unchecked"})
-        public ListComparison(@Nonnull Type type, @Nonnull List comparand) {
+        public ListComparison(@Nonnull Type type, @Nonnull List<?> comparand, boolean ignoreNulls) {
```

Comment:
I probably missed it but is there a place where we actually set `ignoreNulls` to `true`?

Reply:
no we don't. we just need it to validate the List correctly

Reply:
could you please revert since this is a `STABLE` API change which is not allowed.